ipfs-cluster/trigger-server/triggerserver.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+    "log"
+    "net/http"
+    "os/exec"
+    "time"
+    "fmt"
+    "io/ioutil"
+    "math/rand"
+    "strconv"
+    "os"
+    "github.com/ipfs/go-ipfs-api"
+    "bytes"
+)
+
+const CONFIG_PATH = "/etc/trigger-server"
+// HTTP API listen port
+const PORT = 8082
+var RUNNER_PATH = os.Getenv("GOPATH") + "src/github.com/ipfs/kubernetes-ipfs/ipfs-cluster/"
+// Runner takes two args
+const RUNNER_NUM_NODES = "5"
```

Comment:
Trigger server should be a kubernetes-ipfs wide application that can be used by other projects than ipfs-cluster. Should have a configuration file where the parameters used by each project can be configured